gdxpds/gdx.py

```diff
@@ -76,174 +76,10 @@
 import numpy as np
 import pandas as pds
 
-logger = logging.getLogger(__name__)
-
-
-# List of numpy special values in gdxGetSpecialValues order
-#                      1E300,  2E300,  3E300,   4E300,               5E300 
-NUMPY_SPECIAL_VALUES = [None, np.nan, np.inf, -np.inf, np.finfo(float).eps]
-
-def convert_gdx_to_np_svs(df,num_dims,gdxf):
-    """
-    Converts GDX special values to the corresponding numpy versions.
+import gdxpds.special as special
```

Comment:
I understand why you moved this functionality to its own module, however, I don't think it is so important as to negate the general principle of not breaking interfaces if you don't have to. Perhaps we get both if you change this import statement to be 

```
from gdxpds.special import (NUMPY_SPECIAL_VALUES, convert_gdx_to_np_svs, ...)
```

?

That is, the functionality will be in gdxpds.special, but perhaps (this should be checked) the interface gdxpds.gdx.convert_gdx_to_np_svs would also work still?

Reply:
I'm happy to add some import statments and test.

Reply:
Added import statement and unremoved some of the functions which are no longer used.